src/wallet/wallet.h

```diff
@@ -88,6 +88,10 @@ enum WalletFeature
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
     FEATURE_HD = 130000, // Hierarchical key derivation after BIP32 (HD Wallet)
+
+    //TODO: FEATURE_HD_SPLIT needs to be bumped to 140000 once we branch of 0.14 //
```

Comment:
We can keep `139999` for 0.14 to make it compatible with wallets created before branching of 0.14 made with this PR. Unsure what's best practice here...